azure-simple/packer/ansible-is/README.md

```diff
@@ -0,0 +1,126 @@
+# WSO2 Identity Server Ansible scripts
+
+This repository contains the Ansible scripts for installing and configuring WSO2 Identity Server and Identity Server Analytics.
+
+## Supported Operating Systems
+
+- Ubuntu 16.04 or higher
+- CentOS 7
+
+## Supported Ansible Versions
+
+- Ansible 2.6.2
+
+## Directory Structure
+```
+.
+├── dev
+│   ├── group_vars
+│   │   └── is.yml
+│   ├── host_vars
+│   │   └── is_1.yml
+│   └── inventory
+├── docs
+│   ├── images
+│   │   ├── Deployment-pattern-1-diagram.png
+│   │   └── Deployment-pattern-2-diagram.png
+│   ├── Pattern1.md
+│   └── Pattern2.md
+├── files
+│   ├── lib
+│   │   ├── amazon-corretto-8.222.10.1-linux-x64.tar.gz
```

Comment:
Do these resources require a specific version of JDK? if that is the case we may need to mention that as well.

Reply:
Will update the tested jar version for this and preferred link.

Reply:
Done